packages/next-swc/crates/next-core/src/next_shared/transforms/swc_ecma_transform_plugins.rs

```diff
@@ -0,0 +1,104 @@
+use anyhow::Result;
+use turbo_tasks_fs::FileSystemPathVc;
+use turbopack_binding::turbopack::ecmascript::OptionTransformPluginVc;
+
+use crate::next_config::NextConfigVc;
+
+#[turbo_tasks::function]
+pub async fn get_swc_ecma_transform_plugin(
+    #[cfg_attr(not(feature = "plugin"), allow(unused))] project_path: FileSystemPathVc,
+    next_config: NextConfigVc,
+) -> Result<OptionTransformPluginVc> {
+    let config = next_config.await?;
+    match config.experimental.swc_plugins.as_ref() {
+        Some(plugin_configs) if !plugin_configs.is_empty() => {
+            #[cfg(feature = "plugin")]
+            {
+                use anyhow::{bail, Context};
+                use turbo_tasks::Value;
+                use turbo_tasks_fs::FileContent;
+                use turbopack_binding::turbopack::{
+                    core::{
+                        asset::Asset,
+                        issue::{IssueSeverity, OptionIssueSourceVc},
+                        reference_type::ReferenceType,
+                        resolve::{
+                            handle_resolve_error, parse::RequestVc, pattern::Pattern, resolve,
+                            PrimaryResolveResult,
+                        },
+                    },
+                    ecmascript::{OptionTransformPluginVc, TransformPluginVc},
+                    ecmascript_plugin::transform::swc_ecma_transform_plugins::{
+                        SwcEcmaTransformPluginsTransformer, SwcPluginModule, SwcPluginModuleVc,
+                    },
+                    turbopack::{resolve_options, resolve_options_context::ResolveOptionsContext},
+                };
+
+                let mut plugins = vec![];
+                for (name, config) in plugin_configs.iter() {
+                    // [TODO]: SWC's current experimental config supports
+                    // two forms of plugin path,
+                    // one for implicit package name resolves to node_modules,
+                    // and one for explicit path to a .wasm binary.
+                    // Current resolve will fail with latter.
```

Comment:
Why does the latter fail?

Reply:
Turbopack does not resolves to an absolute path not related to project root.